rcon/recent_actions.py

```diff
@@ -79,14 +88,16 @@ def update_tks(rcon: Rcon, log: StructuredLogLineWithMetaData):
     killer = cached_actions[killer_steam_id]
     victim = cached_actions[victim_steam_id]
 
+    killer.player_name = killer_name
     killer.last_tk_victim_steam_id_64 = victim_steam_id
-    killer.last_victim_name = victim.player_name
+    killer.last_tk_victim_name = victim_name
     killer.last_tk_victim_weapon = weapon
-    killer.player_name = killer_name
 
+    victim.player_name = victim_name
     victim.last_tk_nemesis_steam_id_64 = killer_steam_id
     victim.last_tk_nemesis_name = killer_name
     victim.last_tk_nemesis_weapon = weapon
-    victim.player_name = victim_name
 
     set_recent_actions(cached_actions)
+    # Refresh the redis cache so it's available for interprocess communication
+    get_recent_actions()
```

Comment:
Sounds like premature optimization to me :D But it's ok.

Reply:
It's actually required if any other process wants to be able to pull it out of the redis cache, it clears on setting, and any other process that was to call it would just get back a totally empty one.

I don't know if we even want it available in any other process, but if another service wants those values for any reason it has to be done like this (or persisted to the database)